scripts/logfetch/entrypoint.py

```diff
@@ -10,6 +10,7 @@
 from grep import grep_files
 from cat import cat_files
 
+VERSION = '0.0.10'
```

Comment:
i know the versioning goofup was my mistake, but can we consolidate on `0.X.0` going forward? best to keep all versions in order, and its more understandable to see version #'s go up instead of down